src/com/missafternoon/budjen/transactions/DefaultLogger.java

```diff
@@ -1,3 +1,5 @@
+package com.missafternoon.budjen.transactions;
```

Comment:
This class isn't specific to `transactions` in my opinion. Same applies to `Logger`. You know what to do ;)